tests/utils.py

```diff
@@ -743,4 +639,5 @@ def get_outbound_queue_entry(data, max_attempts=10, delay=0.5):
             if not outbound_queue_entry:
                 query_count += 1
                 sleep(delay)
+    conn.close()
```

Comment:
conn.close is not needed as we use the `with` block in line 631

Reply:
fixed